.gitignore

```diff
@@ -11,6 +11,8 @@ moc_*.cpp
 ui_*.h
 qrc_*
 default/
+tests/mapreader/mapreader
+tests/staggeredrenderer/staggeredrenderer
```

Comment:
Note that an outdated entry `tests/test_mapreader` is already in this file below. Please put these lines there instead.

I think it's fine to include this for now, but we can clean up this stuff a lot once the qmake projects are out (#1484), since Qbs always puts build artifacts in a separate directory.